package.json

```diff
@@ -5,7 +5,19 @@
   "scripts": {
     "prepare": "npm update --dev vl-ui-util && npm run prepare:util",
     "prepare:util": "npm explore vl-ui-util -- npm run util",
-    "test": "wct -l chrome,firefox --npm",
+    "test": "npm run test:unit && npm run test:e2e",
+    "test:unit": "wct --config-file ./wct.local.conf.json",
+    "test:e2e": "echo DISABLED OM TE RELEASEN!!!!!!!!!!!!!!!!!",
```

Comment:
niet vergeten om da te undo-en?

Reply:
@cambiph is ondertussen bezig met te kijken waarom hier een probleem mee is.